### DISCUSSION TITLE
Any plans for PopoverAPI?

### DISCUSSION THREAD

**Discussion by Insiderto** [floating-ui/floating-ui#3096] [Q&A] [answered]
I'm curious if there are any plans to move towards [PopoverAPI](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API). I realize right now the API has limited availability and is not yet ready to be used in most browsers, but going forward, are there any plans to move to a more native API or will that not be possible?

**Comment by FezVrasta** [comment, marked as answer]
Please see https://github.com/floating-ui/floating-ui/discussions/2196
